tests/plugins/org.polarsys.capella.test.projection.ju/src/org/polarsys/capella/test/projection/ju/FC2FSTest.java

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2020 THALES GLOBAL SERVICES.
```

Comment:
2022 - please also update the other files where you committed changes

Reply:
Done